code/modules/projectiles/guns/ballistic/bow.dm

```diff
@@ -0,0 +1,102 @@
+/obj/item/gun/ballistic/bow
+	name = "bow"
+	desc = "A sturdy bow made out of wood and reinforced with iron."
+	icon_state = "bow_unloaded"
+	item_state = "bow"
+	var/icon_state_loaded = "bow_loaded"
+	var/icon_state_firing = "bow_firing"
+	var/item_state_loaded = "bow"
+	var/item_state_firing = "bow"
+	fire_sound = "bowshot" //
+	mag_type = /obj/item/ammo_box/magazine/internal/bow
+	//var/flags = HANDSLOW //
+	weapon_weight = WEAPON_HEAVY  //
+	var/draw_sound = 'sound/weapons/draw_bow.ogg'
+	var/ready_to_fire = 0
+	var/slowdown_when_ready = 2
```

Comment:
it's already got an implemented slowdown, this makes it worse. I think it'll make it feel really bad.

Reply:
¿Want a gif to see how it works in game?

Reply:
https://github.com/BadDeathclaw/TG-Claw/blob/master/code/modules/projectiles/gun.dm#L77

much like how i fucked up here and should have made it

slowdown = slowdown + (w_class / 3) we need something similar here. I should go back and fix that too.